src/docs/transforms.md

```diff
@@ -1,8 +1,10 @@
 ---
 eleventyNavigation:
-  parent: Configuration
+  parent: Template Features
   key: Transforms
-  order: 10
+  title: Postprocess Content
+  excerpt: Use Transforms to post-process content after Eleventy builds but before it outputs (usually by writing to disk).
```

Comment:
Could be just me but this sentence reads slightly confusing. I can guess how it is meant, but the „(usually by writing to disk)” could also refer to the „Transforms to post-process content” part.

I'd rephrase it to „(by writing to disk)” properly.

Reply:
You’re right that’s an unnecessary level of detail. Removed the last part altogether